gnovm/tests/file.go

```diff
@@ -592,17 +600,19 @@ func (tb *testBanker) SendCoins(from, to crypto.Bech32Address, amt std.Coins) {
 }
 
 func (tb *testBanker) TotalCoin(denom string) int64 {
-	panic("not yet implemented")
+	return tb.totalCoin[denom]
 }
 
 func (tb *testBanker) IssueCoin(addr crypto.Bech32Address, denom string, amt int64) {
 	coins, _ := tb.coinTable[addr]
 	sum := coins.Add(std.Coins{{denom, amt}})
 	tb.coinTable[addr] = sum
+	tb.totalCoin[denom] += amt
```

Comment:
do we need to check overflow/underflow, as what coins.Add/coins.Sub already did.

Reply:
https://github.com/gnolang/gno/pull/2671/commits/d32ed5f929f9607c0d5730ce10c8c6133510f1d1